docs/source/architecture/slashing.rst

```diff
@@ -101,13 +101,13 @@ Penalty Scenarios:
    
 * *Scenario 3*: Staker incurs penalty calculated to be worth **400 tokens**:
 
-    The difference from the previous scenario is that should also decrease locked tokens in the current period. At the first step the 2nd sub stake is reduced to 100 tokens. Next step - adjustment for the next period. The shortest sub stake still the same - the 2nd. And we need to deacrese it from 100 to 0 only for the next period. Will be the same if we change duration of the 2nd sub stake from 2 periods to 1 and the other sub stakes remain unchanged.
+    The difference between this and the previous scenario is that the current period's sum of locked tokens is also reduced. The first step is to reduce the 2nd sub-stake to 100 tokens. Then, the next period is adjusted – the shortest sub-stake is still the 2nd – and it is reduced from 100 to zero for the next period. Notably, this would have the same result if we changed the duration of the 2nd sub-stake from 2 periods to 1 and the other sub-stakes remained unchanged.
```

Comment:
I think the way we're explaining this is very confusing. We have 1,000 initial tokens and in this and the last example, we slash X amount but the `Result` displays an amount (1,000 - X + 100) which I think will confuse everyone

Reply:
Could you point where is 1,000 - X + 100?

Reply:
@vzotova , it's the fact that we start with 1000 tokens, get slashed 400 tokens, but there are 700 (500 + 100 + 100) tokens accounted for in our `Result` statement.

It's unclear why the 2nd sub-stake still ends up with 100 tokens...? even though the full 200 tokens are slashed form it.

Reply:
Is a timing thing, where the 3rd sub-stake has 100 tokens that won't be locked until the next period - so the same 100 tokens are instead staked for 1 period as part of the 2nd sub-stake?

Reply:
Yep, both current and next period will have 600 locked tokens, 
I was counting on the picture to help explain with this moment

Reply:
We'll need to explain this timing thing a more clearly. The diagram helps, but still needs a solid explanation as a lead in.

Reply:
For me it's difficult to rephrase this, too many time I work with slashing :)
Also keep in mind that ideally user will read "slashing" after "sub-stakes"

Reply:
Can we add something general to the beginning explanation of our slashing methodology?

Basically what I've gleaned from our discussion is that we make every attempt to limit idle "not yet locked" tokens after slashing; these idle tokens will be used in earlier periods before their original lock time, to supplement a slashed locked stake to attempt to keep the pre-slashed total locked stake the same for earlier periods.

Assuming the above is correct, perhaps we can add something like:

```
Once a stake is slashed, tokens that are not currently locked but are set to be locked during future periods, i.e. currently "idle", can be used to replenish/supplement a slashed stake in an attempt to maintain the original pre-slashed total locked tokens for a period.  These "idle" tokens can only be used in periods before their starting lock period, and if necessary, a separate sub-stake is created to facilitate this action.
```

In our scenarios that involve the timing situation above (namely Scenario 2 & 3), we should indicate the available "idle" tokens that are then used to supplement slashed stakes, and the new sub-stake if created.

Reply:
I'm so confused with this explanation, I found somehow wrong some ideas from it. 
Next try :)
First I whant to point that stake is time function

I'll try to explain abstraction and how I was thinking when wrote code:
we have initial figure that consists of several sub-stakes + unlocked tokens, when we slash we move imaginary line from the top to bottom and this line "destroys" or "presses" sub-stakes starting from the shortest, but we can have sub-stakes only as rectangles so when imaginary line pushes - whole sub-stake becomes smaller

next step is restoring some sub-stakes where possible to minimize damage (time-issues in explanation come from this part): we still have this imagimary line that destroyed some sub-stakes and stoped but we will add some small sub-stakes which do not cross this line and restores some locks - this source for 4th sub-stake in 2nd scenario. 
In 3rd scenario imaginary line works as just press, 3rd sub-stake "stronger" then "second" (because longer) and 3rd "cuts" part of 2nd under the press. 

Not sure that'll help :)

Reply:
I think we are in line with the understanding, seems like just figuring out a wording that works 😅 .

In Scenario 2: 
the 2nd sub-stake is reduced from 200 to 100 tokens, and the 100 "idle" tokens from the 3rd sub-stake and used to create the 4th sub-stake for staking for 1 period.

In Scenario 3:
The 2nd sub-stake is emptied from 200 to 0 tokens, and the 100 "idle" tokens from the 3rd sub-stake are used in the 2nd sub-stake directly (since it has no tokens) for 1 period.

Correct?

Reply:
can't say that is correct 
There is no such thing like "idle" tokens
Maybe this picture will show why I'm so confused:
```
            stake
            ^
            |
         700+-----+----+
            |     | 3rd|
         600|     +----+-------------+
            |    2nd   |     3rd     |
         500+----------+-------------+----------+
            |                                   |
            |               1st                 |
            |                                   |   period
            +-----------------------------------+--->
```

Reply:
I've added more diagrams in slashing doc